### DISCUSSION TITLE
Calculating stats for this user. Check back later

### DISCUSSION THREAD

**Discussion by Havasiz** [anuraghazra/github-readme-stats#2531] [Q&A] [answered]
Exception: Query 'https://wakatime.com/api/v1/users/current/all_time_since_today?api_key=***' failed to run by returning code of 202: {'data': {'decimal': '0.00', 'digital': '0:00', 'is_up_to_date': False, 'percent_calculated': 0, 'range': {'end': '2023-02-19T22:59:59Z', 'end_date': '2023-02-19', 'end_text': 'Today', 'start': '2023-02-18T23:00:00Z', 'start_date': '2023-02-19', 'start_text': 'Today', 'timezone': 'Europe/Paris'}, 'text': '0 secs', 'timeout': 15, 'total_seconds': 0}, 'message': 'Calculating stats for this user. Check back later.'}

Hey so I created my wakatime account 1 hour ago and I tried setting up the readme thing but I get this error? Does this mean I need to wait a couple of days and re-try again?

**Comment by Zo-Bro-23** [comment, marked as answer]
Can you please share your wakatime username with us? If it still isn't working, I'll look into it.

Edit: It looks like wakatime needs some time after creating a new user. It should hopefully be fixed now.

**Reply by Havasiz** [reply]
It actually got fixed after waiting for like a day. Thanks for the answer tho!

**Comment by beatrizsmerino** [comment]
Hello!!
I'm having the same problem, my wakatime username is [@beatrizsmerino](https://wakatime.com/@beatrizsmerino)

https://github.com/beatrizsmerino/beatrizsmerino/actions/runs/6918946818/job/18821734503

![Captura de pantalla 2023-11-19 a las 7 56 19](https://github.com/anuraghazra/github-readme-stats/assets/14045148/9b1ded29-dcd3-40fa-8414-ecb67274180a)

![Captura de pantalla 2023-11-19 a las 7 58 56](https://github.com/anuraghazra/github-readme-stats/assets/14045148/52ab0e27-d1c5-48ab-a54d-4991fbd299f0)

**Reply by qwerty541** [reply]
Hey, @beatrizsmerino! Looks like you're asking question in wrong repository since your workflow uses https://github.com/anmol098/waka-readme-stats.

**Reply by beatrizsmerino** [reply]
I noticed that the previous comment was about wakatime stats and that's why I wrote here.
